lib/subscribers/platformatic-kafka/index.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+// TODO: add test to cover once this is resolved https://github.com/platformatic/kafka/issues/155
+
+const recorder = require('#agentlib/metrics/recorders/generic.js')
+const {
+  TracingChannelSubscription,
+  TracingChannelSubscriber
+} = require('../tracing-channel-subscriber.js')
+
+const connectsSub = new TracingChannelSubscription({
+  channel: 'tracing:plt:kafka:connections:connects',
+  start(event) {
```

Comment:
This segment is getting named as `Truncated/connect` as well because segment is never touched